core/include/traccc/utils/logging.hpp

```diff
@@ -0,0 +1,40 @@
+/** TRACCC library, part of the ACTS project (R&D line)
+ *
+ * (c) 2021-2022 CERN for the benefit of the ACTS project
+ *
+ * Mozilla Public License Version 2.0
+ */
+
+/*
+ * This file was copied from the ACTS repository, which is also available
+ * under the Mozilla Public License Version 2.0
+ */
```

Comment:
I guess this is a leftover? Just our normal header should be appropriate for the current state of the file I think.

Reply:
Well spotted! Removed.